linkerd/install.go

```diff
@@ -103,13 +109,23 @@ func (linkerd *Linkerd) applyHelmChart(version string, namespace string, isDel b
 	// Create namespace in which the installation was requested - Both
 	// Helm and Linkerd to are too picky about this
 	createHelmNS(linkerd.MesheryKubeclient, namespace, "linkerd2")
```

Comment:
Why are we doing this manually? Why not just set `CreateNamespace: true` in the meshkit function?

Reply:
I also thought the same thing, I haven't written this piece of code.

Reply:
removing this

Reply:
Actually I just remembered this comment you had made
		// CreateNamespace: true, // Don't use this => Linkerd NS has "special" requirements

Reply:
I guess , some extra annotations are needed on the namespace. Which you are doing inside this function